feat(components)!: create NavBar component and reviews page

knakamura13 opened the pull request:
Created a NavBar utilizing the MUI AppBar component. The NavBar is rendered on every page by
_app.tsx (line 31).

Also created a "reviews" page. Our main focus will be the content on the index page (Courses) and this new page (Reviews).

BREAKING CHANGE: New components and modifications to the app container and index.tsx files.

re #42

lumpygator commented on pages/reviews.tsx:
```diff
@@ -0,0 +1,38 @@
+import ProTip from '../src/ProTip'
+import Copyright from '../src/Copyright'
+
+const About: NextPage = () => {
```
Should `About` be `Reviews` here?

knakamura13 replied:
Probably, but I"m not sure because I just duplicated the "about" page and renamed the file "reviews." 
I might just delete the reviews stuff from this PR.

knakamura13 commented:
> can you add that to storybook rather than into this directory?

How do I do that? Do you want me to just move the `NavBar` file to `/storybook-static`?

ctran4347 commented:
> > can you add that to storybook rather than into this directory?
> 
> How do I do that? Do you want me to just move the `NavBar` file to `/storybook-static`?

In `src/stories`. The `storybook-static folder` is going to be removed, I was using that to deploy to Vercel but we are deploying via Chromatic. Refer to the [storybook documentation](https://storybook.js.org/docs/react/writing-stories/introduction) when creating stories.

knakamura13 commented:
> In `src/stories`. The `storybook-static folder` is going to be removed, I was using that to deploy Storybook to Vercel but we are deploying via Chromatic. Refer to the [Storybook documentation](https://storybook.js.org/docs/react/writing-stories/introduction) when creating stories.

Oh, that's where I had placed NavBar but I just moved it to `storybook-static`.

knakamura13 commented:
I created the storybook, moved the component to its correct folder (`src/stories`), and removed the reviews page and its references from this PR. That marks the completion of all of the requested changes.

herman-rogers reviewed (approved):
lgtm!

ctran4347 reviewed (approved):
Looks good to me too!
